src/Renderer/ThreeExtended/GLTFLoader.js

```diff
@@ -0,0 +1,2219 @@
+// This file is copy pasted from THREE
```

Comment:
Not in an easy way unfortunately. It's a quick fix for the release. If I find something better, I'll do a PR after the release, but the most important is to have a npm package that works for this release. WDYT?